pkg/agent/route/route_windows.go

```diff
@@ -157,18 +158,35 @@ func (c *Client) Reconcile(podCIDRs []string) error {
 	if err != nil {
 		return err
 	}
-	for dst, rt := range routes {
-		if reflect.DeepEqual(rt.DestinationSubnet, c.nodeConfig.PodIPv4CIDR) {
+	for i := range routes {
+		// Retrieve the route entries that use global unicast IP address as the destination. "GetNetRoutesAll" also
+		// returns the entries of loopback, broadcast, and multicast, which are added by the system when adding a new IP
+		// on the interface. Don't remove them since removing those route entries might introduce the host networking issues.
+		if routes[i].DestinationSubnet.IP.IsGlobalUnicast() {
+			continue
+		}
+		// When configuring an IP address to an interface on Windows, three route entries will be automatically added.
+		// For example, if the IP address is 10.10.0.1/24, the following three routes will be created:
+		// Network Destination   Netmask          Gateway  Interface  Metric
+		// 10.10.0.1             255.255.255.255  On-link  10.10.0.1  281
+		// 10.10.0.0             255.255.255.0    On-link  10.10.0.1  281
+		// 10.10.0.255           255.255.255.255  On-link  10.10.0.1  281
+		// The host (10.10.0.1) and broadcast (10.10.0.255) routes should be ignored since they are not supposed to be
+		// managed by Antrea. We can ignore them by comparing them to the calculated broadcast IP.
+		if routes[i].DestinationSubnet.IP.Equal(iputil.GetLocalBroadcastIP(routes[i].DestinationSubnet)) {
+			continue
+		}
+		if reflect.DeepEqual(routes[i].DestinationSubnet, c.nodeConfig.PodIPv4CIDR) {
```

Comment:
would it make sense to use `IPNetEqual` here?

Reply:
Yes, updated.